### PR TITLE
fix(attn): pick the multi-block SWA layout from the device instead of env var

### PR DESCRIPTION
**Pull request by rebel-eunji**
### 🚀 Summary of Changes

Remove VLLM_RBLN_USE_MULTI_BLOCK_ATTN.

A sliding-window layer now takes upstream's `SlidingWindowSpec` when current_platform.is_cr13() and the RBLN shift-kernel spec otherwise, so the appending kernel is used exactly on the NPU that carries it.

---

### 📌 Related Issues / Tickets

* Resolves #
* Related to #

---

### ✅ Type of Change

* [ ] 🚀 Release (`release`)
* [ ] ✨ Feature (`feature`)
* [ ] 🧠 Model support (`model`)
* [ ] 🧬 Core engine changes (`core`)
* [ ] 🛠 Bug fix (`fix`)
* [ ] ⚙️ Performance improvement (`perf`)
* [ ] 🔁 Refactor or code cleanup (`refactor`)
* [ ] 📄 Documentation (`docs`)
* [ ] ❓ Other (`other`): please describe

---

### 🧪 How to Test

1. Run `...`
2. Verify output: `...`
3. Edge case tested: `...`

---

### 📸 Screenshots / Logs (if applicable)

---

### 📋 Checklist

* [ ] PR title follows Conventional Commits format
* [ ] This PR is linked to an existing issue
* [ ] The test method is described, and the expected result is clearly stated
* [ ] Relevant documentation has been updated (if applicable)

---

### 💬 Notes

---

### CODE REVIEW

**Review by rebel-jonghewk — APPROVED** [review]

File: vllm_rbln/envs.py [hunk 1]
```diff
@@ -414,6 +404,19 @@ def use_auto_port() -> bool:
...
+        "VLLM_RBLN_SPECIALIZE_MOE_DECODE",
+        "VLLM_RBLN_USE_MOE_TOKENS_MASK",
+        "VLLM_RBLN_DISPATCH_ALL2ALL",
+        "VLLM_RBLN_COMBINE_ALL2ALL",
+        "VLLM_RBLN_DECODE_BATCH_BUCKET_STRATEGY",
+        "VLLM_RBLN_DECODE_BATCH_BUCKET_MIN",
+        "VLLM_RBLN_DECODE_BATCH_BUCKET_STEP",
+        "VLLM_RBLN_DECODE_BATCH_BUCKET_LIMIT",
+        "VLLM_RBLN_DECODE_BATCH_BUCKET_MANUAL_BUCKETS",
+        "VLLM_RBLN_USE_W8A8",
```
Comment (rebel-eunji) [inline vllm_rbln/envs.py]:
```suggestion
```

File: vllm_rbln/v1/worker/utils.py [hunk 2]
```diff
@@ -332,6 +332,12 @@ def minimum_kv_blocks(vllm_config: VllmConfig, cfg: KVCacheConfig) -> KvMinimum:
         one_request += cdiv(
             spec.max_memory_usage_bytes(vllm_config), spec.page_size_bytes
         )
+        # Unwrap only after the block count above, which the wrapper answers for
+        # the whole group. It is not a SlidingWindowSpec subclass, so both the
+        # lookup and the guard below would miss it; its members are all one
+        # type, so any of them carries the group's admission cap.
```
Comment (rebel-eunji) [inline vllm_rbln/v1/worker/utils.py]:
```suggestion
```

File: vllm_rbln/v1/worker/utils.py [hunk 3]
```diff
@@ -332,6 +332,8 @@ def minimum_kv_blocks(vllm_config: VllmConfig, cfg: KVCacheConfig) -> KvMinimum:
         one_request += cdiv(
             spec.max_memory_usage_bytes(vllm_config), spec.page_size_bytes
         )
+        if isinstance(spec, UniformTypeKVCacheSpecs):
+            spec = next(iter(spec.kv_cache_specs.values()))
```
Comment (rebel-eunji) [inline vllm_rbln/v1/worker/utils.py:336]: It's a safe logic for some SWA models in the future.

> An all-sliding-window model is safe while every layer's spec is identical — the group gets the bare SlidingWindowSpec. The wrapper appears only when an all-SWA model's layers differ in hidden size, or for a DeepseekV4-style hybrid. gpt-oss is a full+SWA hybrid, so both groups arrive bare and never hit this.

**Review by rebel-cjlee — APPROVED** [review]
